osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -42,13 +43,13 @@ public DrawableJudgement(Judgement judgement, DrawableHitObject judgedObject)
         }
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colours)

```

Comment:
The colours should come from the `OsuColour` palette. Rather than the attributes, have a switch statement here over all judgements, and take their colours from the palette.